kythe/go/storage/pebble/pebble.go

```diff
@@ -122,6 +124,16 @@ func (p *pebbleDB) Get(ctx context.Context, key []byte, opts *keyvalue.Options)
 	return value, nil
 }
 
+func (p *pebbleDB) CompactRange(r *keyvalue.Range) error {
+	start := []byte{byte(0)}
+	end := []byte{byte(255)}
```

Comment:
what does this default value for start/end mean?? i'm struggling to parse how pebble uses these, but it seems like you should want this to mean "all keys"--i guess it's surprising to me that byte(255) would be lexicographically last?

Reply:
Fixed.